data/data/gcp/variables-gcp.tf

```diff
@@ -115,3 +115,8 @@ variable "gcp_image_licenses" {
   default = []
 }
 
+variable "gcp_root_volume_kms_key_link" {
+  type       = string
+  description = "The GCP self link of KMS key to encrypt the volume"
+  default = null
```

Comment:
column alignment is off and causing `tf-fmt` to fail

Reply:
> column alignment is off and causing `tf-fmt` to fail

weird, tf-fmt seems to be suggesting different alignment for this file than the other two:
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/4397/pull-ci-openshift-installer-master-tf-fmt/1329497726143959040/build-log.txt